src/components/v5/common/CompletedAction/partials/rows/DecisionMethod.tsx

```diff
@@ -18,13 +18,19 @@ const DecisionMethodRow = ({
   isMultisig,
 }: DecisionMethodRowProps) => {
   const getRowContent = () => {
+    const textContent: Record<DecisionMethod, string> = {
+      [DecisionMethod.MultiSig]: 'Multi-Sig',
+      [DecisionMethod.Permissions]: 'Permissions',
+      [DecisionMethod.Reputation]: 'Reputation',
```

Comment:
Thanks for the change @davecreaser - but this will just give us the hard coded string, it will need to be ran through `formatText` or something to use with react-intl.

Here is where it is setup within the action form for comparison: `/src/components/v5/common/ActionSidebar/partials/DecisionMethodField/DecisionMethodField.tsx`

Reply:
Ah great call! I blame Mondays. Thanks for the file link too, helped speed things up. I've changed it (and the file you linked to) to use the same translation string and added it to the en18 file.